src/simplewallet/simplewallet.cpp

```diff
@@ -7331,8 +7331,12 @@ bool simple_wallet::show_transfer(const std::vector<std::string> &args)
       if (pd.m_unlock_time < CRYPTONOTE_MAX_BLOCK_NUMBER)
       {
         uint64_t bh = std::max(pd.m_unlock_time, pd.m_block_height + CRYPTONOTE_DEFAULT_TX_SPENDABLE_AGE);
+        uint64_t last_block_reward = m_wallet->get_last_block_reward();
+        uint64_t suggested_threshold = last_block_reward ? (pd.m_amount + last_block_reward + 1) / last_block_reward : 0;
```

Comment:
Shouldn't it be `(pd.m_amount + last_block_reward - 1) / last_block_reward` instead, as done in the RPC version?